borrowing/views.py

```diff
@@ -10,5 +8,19 @@
 )
 
 
-class BorrowingViewSet():
-    pass
+class BorrowingListView(generics.ListCreateAPIView):
+    queryset = Borrowing.objects.select_related("book", "user")
+    serializer_class = BorrowingSerializer
+
+
+class BorrowingDetailView(generics.RetrieveAPIView):
+    queryset = Borrowing.objects.select_related("book", "user")
+    serializer_class = BorrowingSerializer
+
+
+class UserBorrowingsListView(generics.ListAPIView):
+    serializer_class = BorrowingSerializer
+
+    def get_queryset(self):
+        user = self.request.user
```

Comment:
use self.request.user.pk to have same results put avoid problem when unathenticated user try to visit this page